docs/rest-api/source/includes/wp-api-v3/reports.md

```diff
@@ -0,0 +1,252 @@
+# Reports
+
+The Reports API allows you to obtain detailed records of transactions and authorizations.
+
+## List transactions
+
+_@since 6.5.0_
+
+Fetch a detailed overview of transactions.
+### Error codes
+
+-   `rest_forbidden` - Unauthenticated
+-   `rest_invalid_param` - Invalid request parameter
+-   `wcpay_server_error` - Unexpected server error
+
+### HTTP request
+
+<div class="api-endpoint">
+	<div class="endpoint-data">
+		<i class="label label-get">GET</i>
+		<h6>/wp-json/wc/v3/payments/reports/transactions</h6>
+	</div>
+</div>
+
+### Optional parameters
+
+- `date_before`: Filter transactions before this date.
+- `date_after`: Filter transactions after this date.
+- `date_between`: Filter transactions between these dates.
```

Comment:
Is date filter for transactions is working?

Reply:
Yes, they do. I just tested it.